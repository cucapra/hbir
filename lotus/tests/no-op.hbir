// This is the closest to "no-op" possible,
//   given the current parser + compiler combo.
// Currently, the parser requires too much info
target{
    memory g[0]{
        size 8G;
        width 8B;
    };

    tile t[0][0];
}

config{
    group tg[0][0] {
        tile target.t[x][y];
    };
}

data{
    t = 0;

    A: int[dim] = block[target.g.x_max] {
        target.g[x];
        global;
        chunked;
        host;
    };

}

code{
}
