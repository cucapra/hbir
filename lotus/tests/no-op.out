#include "bsg_manycore.h"
#include "bsg_set_tile_x_y.h"
#define t 0

int A[dim] __attribute__ ((section (".dram")));
//empty dmaps list

}

